doc/changelog/02-specification-language/10441-static-poly-section.rst

```diff
@@ -0,0 +1,11 @@
+- The universe polymorphism setting now applies from the opening of a section.
+  In particular, it is not possible anymore to mix polymorphic and monomorphic
+  definitions in a section when there are no variables nor universe constraints
+  defined in this section. This makes the behaviour consistent with the
+  documentation. (`#10441 <https://github.com/coq/coq/pull/10441>`_,
+  by Pierre-Marie Pédrot)
+
+- The Section vernacular command now accepts the "universes" attribute. In
```

Comment:
```suggestion
- The :cmd:`Section` vernacular command now accepts the "universes" attribute. In
```

Also, could you document this somewhere (else) than just the changelog?

Reply:
Hmm, the `universes` attribute is not even documented. There is a mention of the deprecated `(mono|poly)morphic` attributes in their respective chapter, but nothing more. Should this go into the universe poymorphism chapter?

Reply:
Is it me or there isn't any Sphinx command to specify an attribute, similar to what `.. flag::` does?